operators/netobserv-operator/0.1.2/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,452 @@
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /manager
+                image: quay.io/netobserv/network-observability-operator:0.1.2
+                imagePullPolicy: Always
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 200Mi
+                  requests:
+                    cpu: 100m
+                    memory: 100Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: netobserv-controller-manager
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: netobserv-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - network observability
+  - ipfix
+  - flow tracing
+  - flows
+  - network
+  - observability
+  links:
+  - name: Project page
+    url: https://github.com/netobserv/network-observability-operator
+  - name: Issue tracker
+    url: https://github.com/netobserv/network-observability-operator/issues
+  - name: Discussion board
+    url: https://github.com/netobserv/network-observability-operator/discussions
+  maintainers:
+  - email: jtakvori@redhat.com
+    name: Joel Takvorian
+  - email: jpinsonn@redhat.com
+    name: Julien Pinsonneau
+  - email: mmaciasl@redhat.com
+    name: Mario Macias
+  - email: ocazade@redhat.com
+    name: Olivier Cazade
+  maturity: alpha
+  provider:
+    name: Red Hat
+    url: https://www.redhat.com
+  version: 0.1.2
+  replaces: netobserv-operator.v0.1.0
```

Comment:
@jotak maybe you want `replaces: netobserv-operator.v0.1.1`

Reply:
yeah I just noticed that, thanks

Reply:
hey @mvalarh , jobs are still running here but they probably fail for the same reason than on k8s-community here: https://github.com/k8s-operatorhub/community-operators/runs/6273680664?check_suite_focus=true

basically, we changed the "app" label of operator workloads, but then the upgrade fails because labels are immutable... 
Do you know what's the options we have? Are we forced to rollback to the previously used labels ?